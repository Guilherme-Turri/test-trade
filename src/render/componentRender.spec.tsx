import { screen } from '@testing-library/react';
import { componentRender } from './componentRender';

describe('componentRender', () => {
  it('renders children componenet', () => {
    const ChildComponent = () => <p data-testid="child">Child</p>;
    const { container } = componentRender(<ChildComponent />);
    const child = screen.getByTestId('child');
    expect(child).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
