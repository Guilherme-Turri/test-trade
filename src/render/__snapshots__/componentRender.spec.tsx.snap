// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`componentRender renders children componenet 1`] = `
<div>
  <p
    data-testid="child"
  >
    Child
  </p>
</div>
`;
